### PR TITLE
Replace deprecated RegistryInterface with Doctrine ManagerRegistry

### PR DESCRIPTION
**Pull request by HugoBroudeur**
## Subject

Since version 1.12 of DoctrineBundle, the `Symfony\Bridge\Doctrine\RegistryInterface` class is deprecated and replaced by `Doctrine\Common\Persistence\ManagerRegistry`.

The change log is available here: https://github.com/doctrine/DoctrineBundle/blob/master/UPGRADE-1.12.md

Since version 2.0 of DoctrineBundle, the class `Symfony\Bridge\Doctrine\RegistryInterface` is removed. Changelog is available here: https://github.com/doctrine/DoctrineBundle/blob/master/UPGRADE-2.0.md

The class `Sonata\TimelineBundle\Spread\AdminSpread` must reflect those change. A bug has been reported here: https://github.com/sonata-project/SonataTimelineBundle/issues/231

I am targeting the `stable` branch, because the change because the old registry implements the new interface.

## Changelog

```markdown
### Added
- Added DoctrineBundle dependency in the `composer.json` 

### Changed
- Changed `Sonata\TimelineBundle\Spread\AdminSpread::__construct` signature
  to use `Doctrine\Common\Persistence\ManagerRegistry` 
  instead of  `Symfony\Bridge\Doctrine\RegistryInterface`
```

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (core23) [inline .idea/.gitignore]: Don't do this please

Comment (core23) [reply .idea/.gitignore]: You should not commit your local IDE files

**Comment by core23** [comment]
Please pick the stable branch. This is not a BC break, because the old registry implements the new interface

**Comment by HugoBroudeur** [comment]
Thank you for reviewing

**Comment by OskarStark** [comment]
This is no BC break? The class is not final 🤨

File: src/Spread/AdminSpread.php [hunk 2]
```diff
@@ -13,11 +13,11 @@
 
 namespace Sonata\TimelineBundle\Spread;
 
+use Doctrine\Common\Persistence\ManagerRegistry;
```
Comment (core23) [inline src/Spread/AdminSpread.php]: We need `doctrine/persistence` as a dependency

Comment (HugoBroudeur) [reply src/Spread/AdminSpread.php]: Indeed, it makes more sense. I went straight to v1.3 of doctrine/persistence and use the non deprecated class Doctrine\Persistence\ManagerRegistry instead.

**Comment by greg0ire** [comment]
Thanks @HugoBroudeur !

**Comment by greg0ire** [comment]
> This is no BC break? The class is not final 🤨

@OskarStark signature compatibility checks (and the LSP) do not apply to the constructor.

**Pull request merged**